requirements.win64.conda.yml

```diff
@@ -17,21 +16,24 @@ dependencies:
   - matplotlib
   - numpy
   - opencv>=4.0
+  - openjpeg>=2.3.0
   - pandas
   - pillow
-  - openjpeg>=2.3.0
   - pip
   - pixman<0.38.0
+  - pygeos
+  - python=3.7
   - pytorch
   - pyyaml>=5.1
   - requests
   - scikit-image
   - scikit-learn>=0.23.2
+  - shapely
+  - shapely
```

Comment:
repetition

Reply:
I can't fix this currently as when I open the file there is only one ther